### ISSUE TITLE
[BUG] Will not compile when using @NoArgsConstructor and @AllArgsConstructor annotations for a class with no attributes

### ISSUE THREAD

**Issue by chinesecooly** [projectlombok/lombok#3333]
When using the @NoArgsConstructor and @AllArgsConstructor annotations for a class with no attributes together, the compiler will prompt for two parameterless constructs

**Comment by sumitverma77** [comment]
Yes, @NoArgsConstructor and @AllArgsConstructor try to generate the same no-arg constructor, which leads to a compilation error because of duplicate constructors.
The best way to avoid the issue is to use only the constructor annotation that is actually needed in such cases.
